scss/components/_title-bar.scss

```diff
@@ -42,7 +42,7 @@ $titlebar-icon-spacing: 0.25rem !default;
 
     @if $global-flexbox {
       display: flex;
-      justify-content: space-between;
+      justify-content: flex-start;
```

Comment:
Interestingly, this component can be manipulated with the flex helper classes. I think title bar should get it's own docs page.